code/modules/jobs/job_types/enclave.dm

```diff
@@ -361,6 +408,7 @@
 	ADD_TRAIT(H, TRAIT_SURGERY_HIGH, src)
 	ADD_TRAIT(H, TRAIT_CHEMWHIZ, src)
 	ADD_TRAIT(H, TRAIT_UNETHICAL_PRACTITIONER, src) // Brainwashing
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/USAeyebot)
```

Comment:
I'd personally give the crafting recipe to all Enclave jobs.

Reply:
No, it can be spammed too much over the radio and it will be annoying, two slots is enough.